src/rkr/util/options.hh

```diff
@@ -32,9 +32,12 @@ namespace options {
   /// Enable file-staging cache
   inline bool enable_cache = true;
 
+  /// Eric: Enable frontier mode
```

Comment:
Is it possible to replace "eric" with "[pash]" in these comments? :) just so that it is easier to track them and associate them with the pash project. At some point we should delete these tags in all comments